src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    static final int MAX_NUM_FIGURES = 5;
+    static final int MAX_NUM_FOR_AREA_X = 20;
+    static final int MAX_NUM_FOR_AREA_Y = 20;
+    static final int MAX_NUM_FOR_AREA_Z = 20;
```

Comment:
```suggestion
    public static final int MAX_NUM_FIGURES = 5;
    pubic static final int MAX_PROPERTY_VALUE = 20;
```
can we have only 1 constant with value 20?